roles/common/tasks/install-upgrade-pypi.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Install and upgrade Python packaging and debugging tools.
+  become: true
+  pip:
+    executable: pip3
```

Comment:
You may just be able to omit the `executable` parameter and the pip3 will be found automatically as default on the host. Depends on platform, naturally, but we already have xenial-specific configs hardcoded as elsewhere (e.g. #50), and the Vagrantfile uses xenial explicitly, so no major concerns there.